test/extensions/bootstrap/wasm/test_data/BUILD

```diff
@@ -140,3 +135,7 @@ wasm_cc_binary(
         "@proxy_wasm_cpp_sdk//:proxy_wasm_intrinsics",
     ],
 )
+
+# TODO(Shikugawa): Use bazel to build this with rewritten Rust test data from
+# https://github.com/istio/envoy/pull/251
+exports_files(["logging_rust.wasm"])
```

Comment:
This is already available, see: https://github.com/envoyproxy/envoy-wasm/pull/611

But if we want to get this in first, and leave the checked in `.wasm` file, then we should probably leave the sources as well, otherwise there is no way to regenerate it.